Tests/Tests/FORMGroupTests.m

```diff
@@ -0,0 +1,13 @@
+@import UIKit;
+@import XCTest;
+
+#import "FORMGroup.h"
+#import "FORMSection.h"
+
+@interface FORMGroupTests : XCTestCase
+
+@end
+
+@implementation FORMGroupTests
```

Comment:
I tried to make some helper methods on FORMGroup but ended up putting everything in `removeSection`

Will improve further afterwards